aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -152,6 +152,7 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
                     ErrorCode.MissingParameter,
                     ErrorCode.ProvisionedIopsNotAvailableInAZFault)
             .withErrorCodes(ErrorStatus.failWith(HandlerErrorCode.NotFound),
+                    ErrorCode.DBClusterNotFoundFault,
```

Comment:
Is it possible that DBInstance resource create throw same exception?

Reply:
I'd say this is a bit too much of an internal implementation disclosure. It certainly can, depending on the specific implementation. This change defines this error code in the `DEFAULT_DB_INSTANCE_ERROR_RULE_SET`, hence should cover both CREATE and UPDATE. Hope it makes sense.